Monika-After-Story/MonikaModDev
# Better input

This will make user experience better. Idea from #5876.

Right now, we have to handle the ability to cancel user input in each use of `renpy.input` (via catching when the user enters `nevermind`). I think it's a bit of ux issue.

The new function was partially *stolen* from RenPy 7. This would allow us to use custom screens for input (not just the `input` one). That's already would be enough (could use a screen with a return button and one w/o it). But to make it even better, I added the ability to pass in kwargs (so we can still have one screen which will handle the both cases).

<details>
  <summary>Example</summary>

  ![Capture](https://user-images.githubusercontent.com/53382877/84494227-c99e3900-acb1-11ea-857a-84eb97cc9a8a.PNG)
  
</details>

I adjusted some topics to use `mas_input`. If you think there's more, feel free to add those.

### Testing
- verify that input (both old and new) still works, and no crashes
- verify that you're able to pass in kwargs to the screen
- verify that you're able to use different screens for input
- check the adjusted topics
